slsim/Sources/SupernovaeCatalog/__init__.py

```diff
@@ -0,0 +1 @@
+from .supernovae_sample import SupernovaeCatalog as SupernovaeCatalog
```

Comment:
Here also, Is it neceassary to use alias?

Reply:
Either aliasing or specifiying in __all__ for the submodule is necessary to uphold linter requirements from ruff. Missed these when switching to __all__ instead of aliasing, thanks for the catch!